csharp/ql/src/utils/modelconverter/project/Program.cs

```diff
@@ -0,0 +1,2 @@
+﻿// See https://aka.ms/new-console-template for more information
+Console.WriteLine("Hello, World!");
```

Comment:
Perhaps let your script do `dotnet new` instead, to avoid having to check in this file.

Reply:
That is a good idea!